src/webui/www/private/index.html

```diff
@@ -5,7 +5,8 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=10" />
     <title>qBittorrent Web UI</title>
-    <link rel="icon" type="image/png" href="images/skin/qbittorrent32.png" />
+    <link rel="icon" type="image/svg+xml" href="images/skin/qbittorrent-tray.svg" />
+    <link rel="alternate icon" type="image/png" href="images/skin/qbittorrent32.png" />
```

Comment:
I didn't found anything about `alternate icon` being a valid entry, at least google didn't return anything meaningful to me.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types
>If there are multiple \<link rel="icon">s, the browser uses their media, type, and sizes attributes to select the most appropriate icon. If several icons are equally appropriate, the last one is used. If the most appropriate icon is later found to be inappropriate, for example because it uses an unsupported format, the browser proceeds to the next-most appropriate, and so on.

Reply:
dropped 'alternate icon' tag, just add new link with 'rel="icon"'